package.json

```diff
@@ -28,7 +28,7 @@
         "access": "public"
     },
     "dependencies": {
-        "@datadog/framepost": "0.2.0"
+        "@datadog/framepost": "https://github.com/DataDog/framepost.git#263c75d619ada57833e0cb6f0d7de386a3ccc136"
```

Comment:
did you mean to check this in? I approved the framepost change if you want to just bump the version instead.

Reply:
I did temporarily, makes it easier for you to test without needing to link everything. Changing to the real version now